introduction.md

```diff
@@ -1 +1,6 @@
 Hi, I'm Shiva! My favorite Code School path was JavaScript. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?